docs/docs/how-tos/nebari-local.md

```diff
@@ -1,4 +1,202 @@
 ---
 id: nebari-local
-title: Deploy Nebari on an existing Kubernetes infrastructure
+title: Deploy Nebari on local machines (for testing)
+description: Deploying Nebari on a local development environment with kind
 ---
+
+Nebari supports a "local testing" mode that helps you develop and efficiently run Nebari on any environment, which can vary from a Virtual Machine, a laptop, or a sizeable multi-node ecosystem.
+
+This installation mode is intended to be used as a debugging tool or, to some extent, as a preview of Nebari functionalities. Thus we do not recommend its use in a production environment.
+It's important to highlight that while it's possible to test most of Nebari with this version, components that are Cloud provisioned such as VPCs, managed Kubernetes cluster, and managed container registries can't be locally tested, due to their Cloud dependencies.
+
+To operate in this mode, Nebari uses "kind" to maintain and deploy the Kubernetes environment where its resources will be integrated with.
+This deployment mode allows quicker feedback loops for development, as well as removes the requisites imposed by using a cloud Kubernetes cluster.
+
+<!-- TODO: Add link to nebari-kubernetes page when ready -->
+
+<!-- Note: If you are looking for installation and deployment instructions for an existing Kubernetes cluster, please visit [Deploy Nebari on an existing Kubernetes infrastructure]. -->
+
+## What is kind?
+
+[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container "nodes".
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+The advantages of using kind are:
+
+- kind supports multi-node (including HA) clusters
+- kind supports building Kubernetes release builds from source
+  - support for make / bash or docker, in addition to pre-published builds
+- kind supports Linux, macOS and Windows
+- kind is a [CNCF certified conformant Kubernetes installer](https://landscape.cncf.io/?selected=kind)
+
+## Compatibility
+
+Nebari integrates kind under the hood by using its Terraform provider and a proper `local` deployment method, which grants native OS compatibility with Linux and MacOS.
```

Comment:
Unfortunately `local` MacOS deployment don't work at the moment. I was working on this a few months ago, see my notes in [this issue](https://github.com/nebari-dev/nebari/issues/1405).

Reply:
Thanks for the note,  updated!